forte/integrals/integrals_psi4_interface.cc

```diff
@@ -369,7 +369,28 @@ void Psi4Integrals::rotate_mos() {
     wfn_->epsilon_b()->copy(eps_a_new);
 }
 
-void Psi4Integrals::build_dipole_ints_ao() {
+psi::SharedMatrix Psi4Integrals::Ca_AO() const {
+    auto aotoso = wfn_->aotoso();
+    auto nao = nso_;
+
+    auto Ca_ao = std::make_shared<psi::Matrix>("Ca_AO", nao, nmo_);
+
+    // Transform from the SO to the AO basis
+    for (int h = 0, index = 0; h < nirrep_; ++h) {
+        auto nso = nsopi_[h];
+        if (nso == 0)
+            continue;
+
+        for (int i = 0, nmo_this = nmopi_[h]; i < nmo_this; ++i) {
```

Comment:
What is the reason we can remove 'resort' from here? Is this the correct way to do the transformation between two sets of Ca?

Reply:
Now I use Pitzer order here. The `resort` was to make the MO dipole integrals in QT order. This was needed because `Ca_subset("AO")` will return QT order. If the goal is simply remove the symmetry, this is the way to go. It might be tempting to use `remove_symmetry` but it actually transforms both rows and columns and not suitable for removing symmetry of `Ca`.

These several lines of the code are repetitively used in DF integrals and I just collect them here for clarity.